test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpLogExporterTests.cs

```diff
@@ -557,6 +562,51 @@ public void CheckToOtlpLogRecordRespectsAttributeLimits()
         var exceptionStackTraceAtt = TryGetAttribute(otlpLogRecord, SemanticConventions.AttributeExceptionStacktrace);
         Assert.Null(exceptionStackTraceAtt);
     }
+    */
+
+    // Remove this when adding back the category, eventid and exception attributes
+    [Fact]
+    public void CheckToOtlpLogRecordRespectsAttributeLimits()
```

Comment:
I think we should keep this test long term. I see you pulled in this test from the refactor I did on #4762.

I think you should refactor the existing test in place like I did on #4762 because it better reflects what the name of the test suggests it does - i.e., validates attributes are truncated. It was weird that this test was conflating testing attribute truncation only for the exception attributes.

Reply:
Done